indigo_app/templates/indigo_app/user_popup.html

```diff
@@ -0,0 +1,23 @@
+<div class="popup-content lh-compact">
+  <div class="card-body">
```

Comment:
This div can be dropped.

Reply:
I've made some additional comments on the content of the popup in the google doc.